test/test_mediawiki.py

```diff
@@ -0,0 +1,65 @@
+import chimedb.core as db
+
+import os
+import pytest
```

Comment:
This is a `pytest` test.  All the other tests are `unittest` tests.  

`pytest` tests are fine with me, probably we should have used that from the beginning, but to get this working, youI think you need to modify the travis file to get this to actually run.

Reply:
Oh, good catch!

Reply:
Also need to add pytest to the dependencies.

Reply:
Hmm, but you don't want pytest to be a dependency of a normal installation of this, do you?

Reply:
I think we never added pytest to the requirements file. But since you asked, I added it as `extras_require`. You can make sure it's installed with `pip install chimedb[tests]`. Maybe this is a good solution for this issue in more of our python repositories.